packages/web/modals/validator-squad.tsx

```diff
@@ -41,26 +46,36 @@ interface ValidatorSquadContentProps {
   isOpen: boolean;
 }
 
+function normalizeUrl(url: string): string {
```

Comment:
I think this won't work if the www is after the http? 

Maybe this would work

```
function normalizeUrl(url: string): string {
  return url.replace(/^https?:\/\/(www\.)?/, "").replace(/\/$/, "");
}
```